x-pack/plugins/observability_solution/slo/server/services/summary_search_client.ts

```diff
@@ -70,14 +70,15 @@ export class DefaultSummarySearchClient implements SummarySearchClient {
       this.logger.error(`Failed to parse filters: ${e.message}`);
     }
 
-    const indices = await getListOfSummaryIndices(this.soClient, this.esClient);
+    const { indices, settings } = await getListOfSummaryIndices(this.soClient, this.esClient);
```

Comment:
This function name is misleading. I think we should find a better name reflecting the fact that it returns the summary indices as well as the slo settings. OR introduce a new function just for the slo settings.